src/BrandedNavBar/MobileMenu.tsx

```diff
@@ -209,10 +208,17 @@ type BaseMobileMenuProps = {
   subtext?: string;
   closeMenu?: Function;
   themeColorObject?: ThemeColorObject;
-  logoSrc?: string;
+  showNulogyLogo?: boolean;
```

Comment:
@haideralsh technically, you are correct that since the `MobileMenu` is exported, it is part of the public API. However, only Eco is using this widget. I'll make it a breaking change if you want though. Let me know!